### ISSUE TITLE
Icons appears as a question . mark

### ISSUE THREAD

**Issue by saimel** [jsmarcus/Iconize#130]
Hi,

After installing this plugin into a new app, all icons appear as a question mark, I also updated versions of Xamarin Forms and Iconize to latest into another app I developed a few months ago and now I have this problem in both apps. Any idea about it??

**Comment by venixsoftwaregeneral** [comment]
Same problem here.
Icons appear as question mark when called in canvas and stay in text "fas-iconname" when used with IconButton.

Xamarin.Forms 3.5.0.169047
Xam.Plugin.Iconize 3.5.0.117

**Comment by ClausElmann** [comment]
Same here

**Comment by jrcpereira** [comment]
Rollback to v3.5.0.114 and it's working.

Duplicate of #126

**Comment by Mr-Pearce** [comment]
> 
> 
> Rollback to v3.5.0.114 and it's working.

v 3.5.0.117 is working fine on android but on ios i get the question mark icons
i also tried downgrading to 3.5.0.114 but it still wont show the proper icons on ios

**Comment by jrcpereira** [comment]
3.5.0.114 is working fine for me on iOS with Forms 3.5.0.169047.
Did you tried to clean up solution and delete bin and obj folders?

**Comment by Mr-Pearce** [comment]
i had Forms  3.6.0.220655
downgrading to 3.5.0.169047 is not working with 3.5.0.114 iconize 

Xam.Plugin.Iconize (114) is installed in my main project(net standard) and two shared project(net standard)

FontAwesome is only installed inside my main project

**Comment by hjavaher** [comment]
Same here, I'm on forms 3.6.0.220655 and the icon shows up as a question mark. Any workarounds other than downgrading Forms?

I installed Xam.Plugin.Iconize and FontAwesome in all three of my projects.

**Comment by saimel** [comment]
I found out a solution in another thread: I downloaded `ttf `files, added as `BundleResource` to my __iOS__ project and everything works out. I've not test it on __Android__.

**Comment by Jsberry92** [comment]
@saimel what version of Forms and Iconize are you running?

**Comment by saimel** [comment]
@Jsberry92  sorry about not specifying versions info, I just forgot it. I'm running these ones:

- `Xamarin.Forms 4.0.0.232914-pre6`
- `Xam.Plugin.Iconize 3.5.0.117`
- `Xam.Plugin.Iconize.Material 3.5.0.117`
- `Xam.Plugin.Iconize.MaterialDesignIcons 3.5.0.117`

I'd like to add I just updated `Xamarin.Forms` from version 3.6.XX three days ago, I tried to solve this issue by using different versions of `Xamarin.Forms` and `Xam.Plugin.Iconize` with no success.

**Comment by Tmaturano** [comment]
@saimel I tried to download the .ttf files from this repository and added inside the Resources folder in the iOS project (as BundleResource), but didn't work.
I'm using `Xamarin.Forms 3.5.0.169047`
`Xaml.Plugin.Iconize 3.5.0.117`
`Xaml.Plugin.Iconize.FontAwesome 3.5.0.117`
Also I tried to downgrade to Iconize 3.5.0.114 but didn't work.

**Comment by saimel** [comment]
@Tmaturano did you add `Fonts` to `info.plist` file ?

**Comment by hjavaher** [comment]
@Tmaturano exactly what @saimel said, I added the fonts(the ones you need, in my case all the ones that said fontawesome) to the info.plist and then added the ttf files from the repo with the same name (rename if necessary) and in the properties of each ttf file make sure the build action is set to BundleRespource (should be default). This did it for me

`<key>UIAppFonts</key>
<array>
    <string>iconize-entypoplus.ttf</string>
    <string>iconize-fontawesome-regular.ttf</string>
    <string>iconize-fontawesome-solid.ttf</string>
    <string>iconize-fontawesome-brands.ttf</string>
    <string>iconize-ionicons.ttf</string>
    <string>iconize-material.ttf</string>
    <string>iconize-meteocons.ttf</string>
    <string>iconize-simplelineicons.ttf</string>
    <string>iconize-typicons.ttf</string>
    <string>iconize-weathericons.ttf</string>
</array>`

**Comment by Tmaturano** [comment]
Yeah, that did the trick, thanks guys

**Comment by Mr-Pearce** [comment]
just tested it and it works 
Xamarin.Forms 3.5.0.169047
Xaml.Plugin.Iconize 3.5.0.117
Xaml.Plugin.Iconize.FontAwesome 3.5.0.117
and putting the fontawesome ttfs inside the ios projcet root

Thanks guys!

**Comment by Jsberry92** [comment]
So this issue seems to fix iOS but breaks Android.
To get Android to work I'm having to role back (on the Android project) Iconize & Iconize.FontAwesome to 3.5.0.114.

I have also tried multiple versions of Xamarin.Forms but nothing works on Android until I roll Iconize back.

Wasn't sure if other people were experiencing this or if I am doing something wrong. But I suppose if that's all I have to do to get Android working for now then that's what I will do.

**Comment by jsmarcus** [comment]
Currently to get Android to work with the latest you will need to target SDK 28. There is a fix for this coming.

**Comment by Jsberry92** [comment]
Thank you @jsmarcus works like a charm!

**Comment by Pirate911** [comment]
I'm having the same issue . It happens on IOS . I tried everything but it didn't work .

Xamarin.Forms 3.5.0.169047
Xam.Plugin.Iconize 3.5.0.117

**Comment by saimel** [comment]
@Pirate911 had you try to download ttf files and add it as Bundle Resource to your iOS project?

**Comment by Pirate911** [comment]
> @Pirate911 had you try to download ttf files and add it as Bundle Resource to your iOS project?

Yes I did all that and I'm certain all required steps are correct .

**Comment by vrheadspace** [comment]
I can confirm after going through each above step, including ttf files, I am still seeing questions marks as well

**Comment by Waaheeda** [comment]
@Chris-Niles @Pirate911  did u find any solution?
